plugins/include/message_const.inc

```diff
@@ -1164,15 +1164,15 @@ enum TracerColor
 	TracerColor_Red,
 	TracerColor_Green,
 	TracerColor_Blue,
-	TracerColor_Yellow,
+	TracerColor_User,
 	TracerColor_Orange,
-	TracerColor_Pink,
+	TracerColor_Coral,
 	TracerColor_Purple,
-	TracerColor_Pink2,
-	TracerColor_Pink3,
-	TracerColor_Pink4,
-	TracerColor_Pink5,
-	TracerColor_DarkBlue
+	TracerColor_Coral2,
```

Comment:
I think we can remove it, because it is just a duplicate of 6 (`TracerColor_Coral`).